src/components/EntryLabel.tsx

```diff
@@ -18,6 +18,13 @@ const StyledLabel = styled.label<{
   display: flex;
   align-items: center;
   color: ${(props) => (props.error ? "#B12424" : "#000000")};
+
+  @media (min-width: 834px) {
```

Comment:
Correct me if I'm wrong, but I'm seeing us use CSS directly but also through styled components in the codebase. Would it be feasible to just switch to one of these two ways?

Reply:
We are using only Tailwind and Styled components. Doing only one of these two ways seems problematic to me.
1. Tailwind only: Certain things require precise values which are not directly provided by Tailwind classes
2. Styled-components only: IMO it adds a lot of painful boilerplate stuff. For every div that has unique CSS, there will need to be a styled component. It doesn't seem feasible to abstract our UI into a reasonable number of styled components.

From my perspective, using both is the easiest to read, maintain, and implement. But I may also be completely wrong about this. Is there a specific reason why you want to switch to one of the two ways?

Reply:
Valid reasons. That's why I asked for feasibility before suggesting we actually go through with the change. 

I remember Dillon and I having a conversation early last semester about using CSS in only one way, and that's why I asked.